src/pages/PeoplePage.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { getPeople } from '../api';
+import { Loader } from '../components/Loader';
+import { PeopleFilters } from '../components/PeopleFilters';
+import { PeopleTable } from '../components/PeopleTable';
+import { Person } from '../types';
+
+export const PeoplePage: React.FC = () => {
+  const { personSlug = '' } = useParams();
+
+  const [people, setPeople] = useState<Person[]>([]);
+  const [hasError, setHasError] = useState(false);
```

Comment:
It's more comfortable to have `errorMessage` state and save there a string error. If it's empty then there are no errors, if it has value then it can be used for any error.